rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1361,6 +1361,10 @@ protected void doTLOAD(){
     protected void doTSTORE(){
         //TODO: Gas cost calculation will be done here and also shared contexts verifications for
         // different types of calls
+        if (program.isStaticCall() && program.getActivations().isActive(RSKIP91)) {
```

Comment:
do you think we should still check for `RSKIP91`? I guess it was only needed when `STATICCALL` opcode was introduced

Reply:
Yes, makes sense. At this height, staticcall would be for sure enabled. I will remove the verification of it.